core/controllers/ingest.py

```diff
@@ -81,6 +81,10 @@ def ingest(body: dict, excel_file: FileStorage) -> tuple[dict, int]:
     try:
         if iv_schema := ingest_dependencies.initial_validation_schema:
             initial_validate(workbook_data, iv_schema, auth)
+
+        if fund == "Pathfinders":
+            return {"detail": "PF initial validation success"}, 200
```

Comment:
I think a comment explaining that this is a temporary stopgap until more of the Pathfinders pipeline is developed could be useful